e2e/main_test.go

```diff
@@ -19,11 +19,13 @@ var client kubernetes.Interface
 // non-configurable for now
 const namespace = "bootkube-e2e-testing"
 
+var (
+	kubeconfig = flag.String("kubeconfig", "../hack/quickstart/cluster/auth/kubeconfig", "absolute path to the kubeconfig file")
```

Comment:
nit: If you need to export kubeconfig as a global. Might be best to still initialize this in TestMain via `flag.StringVar(&kubeconfig...` Just keeps all our init functionality in once place happening in deterministic order. I think either way it will be correct though.